docs/source/usage/plugins/png.rst

```diff
@@ -100,14 +100,12 @@ Since an adequate color scaling is essential, there several option the user can
 
    // normalize EM fields to typical laser or plasma quantities
    //-1: Auto: enable adaptive scaling for each output
-   // 1: Laser: typical fields calculated out of the laser amplitude
+   // 1: Laser: [outdated]
    // 2: Drift: [outdated]
    // 3: PlWave: typical fields calculated out of the plasma freq.,
    // assuming the wave moves approx. with c
    // 4: Thermal: [outdated]
-   // 5: BlowOut: typical fields, assuming that a LWFA in the blowout
-   // regime causes a bubble with radius of approx. the laser's
-   // beam waist (use for bubble fields)
+   // 5: BlowOut: [outdated]
```

Comment:
why is blowout also outdated? 
Couldn't we derive its assumed value from incident field ?

Reply:
Blow out depends on the laser profiles too, there is no incident field equivalent available. 
IMO this is no problem, no example was used it so the code was not tested anyway and png's are only helpers and real visualisation should be done via other tools.

Reply:
Okay - if no one used it, that should be fine. Otherwise it just depends on a_0, which we set in both the old laser and the new incident field.